### DISCUSSION TITLE
Should we release 0.69.3?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#27] [Patches]
## Released
For 0.69.4, please refer to: https://github.com/reactwg/react-native-releases/discussions/28

---

Current Release: [0.69.2](https://github.com/facebook/react-native/releases/tag/v0.69.2)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by ken0nek** [comment]
I'm sure you already know but just for reference

https://github.com/facebook/react-native/pull/34214

**Reply by kelset** [reply]
yeah - we are aware; the PR is already merged and it's directly against the 0.69 branch so it's already cherry-picked basically :)
